magento/magento2
# Add regions for Ukraine

### Description (*)
Add regions for Ukraine

### Related Pull Requests


### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Ukraine regions.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#35187: Add regions for Ukraine
